pipelines/build/devkit/Tools.gmk.patch

```diff
@@ -34,11 +30,12 @@ index 187320ca2..9f1454d5e 100644
    LINUX_VERSION := Fedora_$(BASE_OS_VERSION)
  else
    $(error Unknown base OS $(BASE_OS))
-@@ -95,16 +113,38 @@ endif
+@@ -95,16 +113,39 @@ endif
  ##########################################################################################
  # Define external dependencies
  
 +# Centos GPG KEYS
++Centos_7_GPG_KEY_noarch := 6341AB2753D78A78A7C27BB124C6A8A7F4A80EB5  #gitleaks:allow
 +Centos_7_GPG_KEY_aarch64 := EF8F3CA66EFDF32B36CDADF76C7CB6EF305D49D6 #gitleaks:allow
 +Centos_7_GPG_KEY_x86_64 := 6341AB2753D78A78A7C27BB124C6A8A7F4A80EB5  #gitleaks:allow
 +Centos_7_GPG_KEY_ppc64le := BAFA3436FC50768E3C3C2E4EA963BBDBF533F4FA #gitleaks:allow
```

Comment:
Are these lines unchanged? I can see the addition of noarch but for some reason github is highlighting all of them in the diff

Reply:
again github file rendering of a "patch" file confusing with PR diff